scripts/local-airflow.sh

```diff
@@ -0,0 +1,20 @@
+target_dag=$1
+
+echo "Creating Airflow instance to run $target_dag..."
+
+set -xue
+
+# TODO(wcromar): enable editable installs of `xlml` instead
+export PYTHONPATH=$PWD
+export XLMLTEST_CONFIGS=$PWD/dags/jsonnet
+export XLMLTEST_SSH_EXTERNAL_IPS=1
+
+export AIRFLOW_HOME=$(mktemp -d)
```

Comment:
What's the `-d` flag here?

Reply:
This makes a directory instead of a file.